synapse/storage/databases/main/ui_auth.py

```diff
@@ -339,3 +337,7 @@ def _delete_old_ui_auth_sessions_txn(
             iterable=session_ids,
             keyvalues={},
         )
+
+
+class UIAuthStore(UIAuthWorkerStore):
+    pass
```

Comment:
Would it make more sense to remove the `UIAuthWorkerStore` and just use `UIAuthStore` everywhere instead?

Reply:
Potentially, though its not the worse to keep the separation for consistency.